pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -1416,6 +1420,8 @@ public void doNamespaceBundleSplit() throws Exception {
 
     @Override
     public void stop() throws PulsarServerException {
-        // do nothing
+        loadReportCacheZk.shutdown();
+        availableActiveBrokers.shutdown();
+        scheduler.shutdown();
```

Comment:
Actually, I mean before scheduling task `scheduler.isShutdown()` in case scheduler is already shutdown.